Formula/dzr.rb

```diff
@@ -0,0 +1,22 @@
+class Dzr < Formula
+  desc "Command-line Deezer.com player"
+  homepage "https://github.com/yne/dzr"
+  url "https://github.com/yne/dzr/archive/refs/tags/230501.tar.gz"
+  sha256 "58bb733fb41e52feff489200f18c376d9e9dda9febff391504a51fe31ac3c278"
+  license "Unlicense"
+  head "https://github.com/yne/dzr.git", branch: "master"
+
+  depends_on "dialog"
+  depends_on "jq"
+  depends_on "mpv"
+  depends_on "openssl@1.1" # to match curl one
+  uses_from_macos "curl"
+
+  def install
+    bin.install "dzr", "dzr-url", "dzr-dec", "dzr-srt", "dzr-id3"
+  end
+
+  test do
+    assert_match "3ad58d9232a3745ad9308b0669c83b6f7e8dba4d", shell_output("DZR_CBC= #{bin}/dzr ! | shasum")
```

Comment:
What exactly are we checking here?

Reply:
this line checks if the web player token (always the same) can still be extracted from a random page at dzr init.

I don't want to compare it as clear text as it may be a protected asset (I'm not an expert but I prefer to play safe)

this extraxt seems to fail on older Linux distributions (including dockerized brew) due to a printf quirk

I'm trying to find a dependency-free and cross platform solutions.

Reply:
> I don't want to compare it as clear text as it may be a protected asset (I'm not an expert but I prefer to play safe)

In docker containers on CI there isn't anything protected.

Reply:
The printed token by might be protected by copyright law. So I prefere to hash it for comparing.

Reply:
see https://en.wikipedia.org/wiki/AACS_encryption_key_controversy